include/spatio_temporal_voxel_layer/measurement_buffer.hpp

```diff
@@ -101,6 +102,10 @@ class MeasurementBuffer
   // Get measurements from the buffer
   void GetReadings(std::vector<observation::MeasurementReading>& observations);
 
+  // enabler setter getter
+  bool isEnabled() const;
```

Comment:
We have this repo set up so functions are all Caps, can you change to `IsEnabled` and `SetEnabled`? also, while you're at it, a `const bool&` I know its a small thing but its for consistency. Also add `(void)` in `IsEnabled(void)`.

All small things